modules/representation_management/app/controllers/representation_management/v0/pdf_generator_2122_controller.rb

```diff
@@ -7,7 +7,16 @@ def create
         form = RepresentationManagement::Form2122Data.new(flatten_form_params)
 
         if form.valid?
-          render json: { message: 'Form is valid' }, status: :created
+          Tempfile.create do |tempfile|
+            tempfile.binmode
+            RepresentationManagement::V0::PdfConstructor::Form2122.new(tempfile).construct(form)
+            send_data tempfile.read,
+                      filename: '21-22.pdf',
```

Comment:
Could a static file name cause collisions when there are many pdf generation attempts at the same time? The `DisabilityCompensationController` uses `file_name = SecureRandom.hex.to_s` for example.

Reply:
I think we're in good shape with just `Tempfile`.  The [Tempfile docs](https://docs.ruby-lang.org/en/3.3/Tempfile.html#class-Tempfile-label-Minor+notes) say:

> Tempfile’s filename picking method is both thread-safe and inter-process-safe: it guarantees that no other threads or processes will pick the same filename.
[Tempfile](https://docs.ruby-lang.org/en/3.3/Tempfile.html) itself however may not be entirely thread-safe. If you access the same [Tempfile](https://docs.ruby-lang.org/en/3.3/Tempfile.html) object from multiple threads then you should protect it with a mutex.

Reply:
I think I missed that this is the name of the downloaded file, not necessarily the temp file